app/messages.py

```diff
@@ -47,6 +47,7 @@
 AUTHORISATION_TOKEN_IS_MISSING = {"message": "The authorization token is" " missing!"}
 DESCRIPTION_FIELD_IS_MISSING = {"message": "Description field is missing."}
 COMMENT_FIELD_IS_MISSING = {"message": "Comment field is missing."}
+USERNAME_FIELD_IS_EMPTY = {"message": "The username field has to be longer than 4 characters and shorter than 26 characters."}
```

Comment:
Could you make the message a f-string (e.g.: `f"The username (...)"`) so you can use the length limits that exist for username. you can follow this PR example: https://github.com/anitab-org/mentorship-backend/pull/584/files#diff-3a3e7c840a0d5a8e57c891b68ff5a3b5R14

Reply:
Okay. I'm on it.